packages/plugins/other/visitor-plugin-common/src/selection-set-processor/base.ts

```diff
@@ -36,7 +41,8 @@ export class BaseSelectionSetProcessor<Config extends SelectionSetProcessorConfi
 
   transformPrimitiveFields(
     _schemaType: GraphQLObjectType | GraphQLInterfaceType,
-    _fields: PrimitiveField[]
+    _fields: PrimitiveField[],
+    _unsetTypes?: boolean
```

Comment:
Needed a way to say _hey, don't care about the actual types, override them to `never`_